### ISSUE TITLE
There should be a merge option to ignore blank line removals

### ISSUE THREAD

**Issue by jrood** [git/git-scm.com#860]
If the contents of a file initially are:

```
one

three
```

and on branch A there is a commit changing it to:

```
one
three
```

and on branch B there is a commit changing it to:

```
one
two
three
```

Normally, if you try to merge A into B (or B into A), git would wonder if you want to remove the second line or add "two" to the second line, but there should be a merge strategy that defaults to the latter in cases where the removed line was a blank line (or a line containing only whitespace) ...something like -Xignore-blank-line-removals

In other words, there should be a merge strategy or option where it is assumed that the removal of blank lines is not something that needs to be kept.

**Comment by sxlijin** [comment]
Hey, this is the issue tracker for the git-scm.com website, not Git itself. If you still want to discuss this issue, you should reach out to the [community](https://git-scm.com/community).
